test/flutter_switch_test.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import '../lib/flutter_switch.dart';
+
+void main() {
+  final alignToggleIndicatorFinder = find.byWidgetPredicate(
+      (widget) => widget is Align && widget.child is Container);
+
+  testWidgets(
+    "displays the toggle indicator on the right if the given value is true",
+    (WidgetTester tester) async {
+      await tester.pumpWidget(
+        _FlutterSwitchTestbed(
+          value: true,
+        ),
+      );
+
+      final align = tester.widget<Align>(alignToggleIndicatorFinder);
+
+      expect(align.alignment, equals(Alignment.centerRight));
+    },
+  );
+
+  testWidgets(
```

Comment:
Maybe we can test that the toggle alignment changes after on tap? I think it would be great to test this in such cases due to related bug we've faced:  

- The initial value is `false`
- The initial value is `true`